src/GRA.Domain.Service/AvatarService.cs

```diff
@@ -431,6 +431,43 @@ public async Task<ICollection<AvatarElement>> GetUserAvatarAsync()
             return await _avatarElementRepository.GetUserAvatarAsync(GetActiveUserId());
         }
 
+        public async Task<Tuple<List<AvatarBundle>,List<string>>> GetUserUnlockBundles()
```

Comment:
I think I already mentioned this but it should be renamed to `GetUserUnlockBundlesAsync()`.

Reply:
Also why is this returning a `List<string>` rather than a `List<bool>`? It actually seems like a `Dictionary<AvatarBundle, bool>()` would eliminate the use of `Tuple` and would make things easier?